ipatests/test_webui/ui_driver.py

```diff
@@ -174,6 +175,15 @@ def load_config(self):
         if 'type' not in c:
             c['type'] = DEFAULT_TYPE
 
+    def generate_csr(self, cn, csr_file):
```

Comment:
Hi @mrizwan93 thanks for your patch. I would recommend to use python-cryptography for this.

Please refer to :
https://cryptography.io/en/latest/x509/tutorial/

Then you can create a module which could be re-usable for all the tests in the future.

Reply:
@Rezney 
Thanks for the review. I am trying to understand if python-cryptography have advantage over using openssl. 
In case of openssl we are not importing any package to the code but for python-cryptography we need to import some of them.
Thanks for the tutorial as well, it will help to replace the code if we agrees upon it.

Reply:
No worries, it is your call. Just suggestion. Not blocker...

Reply:
Hello @mrizwan93. I think that it would be better NOT to use some sub-processes as you do there but to use available libraries and create modul as @Rezney proposed. What do you think @flo-renaud ?

Reply:
replacing subprocess makes sense. Thanks @Rezney @Tiboris